change/@fluentui-react-theme-7375db93-6862-41b1-85aa-7312244fca16.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
Can you please merge `master`? This should be also `prerelease` as I remember...

Reply:
You're right, I've changed it!